client/drum-pad.html

```diff
@@ -1,13 +1,15 @@
 <div ng-controller="drumPad" class="page-content">
 	<div class="drum-pad-container">
 		<div class="controls">
-			<img
+			<div
 				class="play-pause-button"
 				ng-class="beatIsPlaying ? 'pause' : 'play'"
-				ng-click="startOrStopBeat()"
-				alt="Play or pause button.">
-			<label>BPM:</label>
-			<input type="number" class="tempo input-" ng-model="tempo" min="60" max="240"></input>
+				ng-click="startOrStopBeat()">
+			</div>
+			<div class="control-variable">
```

Comment:
control-variable should be a container for each of the inputs that will go there. There will be one for tempo, and plans for a number of steps variable, and possibly a time signature one. 

Do you mainly have issue with the name, or with the new class?